config/kraft/server.properties

```diff
@@ -31,22 +31,23 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will be equal to the value of
```

Comment:
Is this still accurate when process.roles=broker,controller?

Reply:
Sorry, I don't get it. Will there be any difference for `process.roles=broker,controller`? Could you let me know. Thanks.

Reply:
What I mean is whether it is allowable for `listeners` to be not configured when `process.roles=broker,controller`. At a minimum, don't we require the controller listener?

Reply:
You are right! Update the description to:

// The address the socket server listens on.
**// This is required for combined role(i.e. process.roles=broker,controller) to configure the listeners for clients and controller**
//   FORMAT:
//     listeners = listener_name://host_name:port
//   EXAMPLE:
//     listeners = PLAINTEXT://your.host.name:9092